pkg/geo/geo.go

```diff
@@ -617,6 +618,37 @@ func S2RegionsFromGeom(geomRepr geom.T, emptyBehavior EmptyBehavior) ([]s2.Regio
 // Common
 //
 
+// Normalize geographical coordinates
+// to spherical coordinates
+func makeValidGeographicalPoint(lat float64, lng float64) (float64, float64) {
+	if math.IsNaN(lat) || math.IsNaN(lng) {
+		return lat, lng
+	}
+	latlng := s2.LatLngFromDegrees(lat, lng)
+	return NormalizeLatitudeDegrees(latlng.Lat.Degrees()),
+		NormalizeLongitudeDegrees(latlng.Lng.Degrees())
+}
+
+// Limit geography coordinates to spherical coordinates
+// by converting geom.T cordinates inplace
+func makeValidGeographyGeom(t geom.T) {
```

Comment:
let's name this `normalizeGeographyGeomT`, and make the comment `normalizeGeographyGeomT limits...` (function name first)